admin-dev/themes/new-theme/js/app/pages/logs/index.js

```diff
@@ -89,5 +90,6 @@ class LogsPage {
 }
 
 $(() => {
+  global.resetSearch = resetSearch;
```

Comment:
no way I can't access resetSearch in my page: did I tell you how much I hate javascript? 

I need a little help here

Reply:
Don't make functions globally accessible, you should avoid inserting behavior directly in your HTML code.

You should inverse the responsibility and place it in your js code.

Bad:
```html
<!-- HTML -->
<button onclick="resetSearch()">Reset</button>
```

Good:
```html
<!-- HTML -->
<button class="reset-search">Reset</button>
```
```js
// JS
// logs/index.js
$('.reset-search').on('click', () => {
  searchForm.resetSearch();
});
```
 instead of imperatively "flag" the control with a class and make